CONTRIBUTING.md

```diff
@@ -2,27 +2,30 @@
 
 Liquid Prep is an open source project and always welcome contributions to the project. You can contribute in the following ways:
 
-* Software and Hardware testing
-* Localization and improvisation of the application
-* Add/Update crop data
-* Enhance watering advise agorithm
-* Pilot testing the solution
-* Improve the documentation
-* Report bug and/or work on existing [issues](https://github.com/Call-for-Code/Liquid-Prep/issues)
-* Request/recommend new features in our [issues](https://github.com/Call-for-Code/Liquid-Prep/issues) or [discussions board](https://github.com/Call-for-Code/Liquid-Prep/discussions)
+- Software and Hardware testing
+- Localization and improvisation of the application
+- Add/Update crop data
+- Enhance watering advise agorithm
+- Pilot testing the solution
+- Improve the documentation
+- Report bug and/or work on existing [issues](https://github.com/Call-for-Code/Liquid-Prep/issues)
+- Request/recommend new features in our [issues](https://github.com/Call-for-Code/Liquid-Prep/issues) or [discussions board](https://github.com/Call-for-Code/Liquid-Prep/discussions)
 
-Please read our [contributing guidelines](#contributing-guidelines).
+Please read our [contributing guidelines](#contributing-guidelines) and founding [Technical Charter](Liquid-Prep-Technical-Charter.pdf).
```

Comment:
@krook is the Technical Charter PDF uploaded in the project? Gives me a 404 error.

Reply:
Ah. Forgot to include the PDF itself. The review process proves its value!

Reply:
Uploaded in the latest commit. Should be good now.

Reply:
Awesome thank you